tests/src/Kernel/PermissionEventTest.php

```diff
@@ -200,6 +204,37 @@ public function permissionEventDataProvider() {
           $group_content_operation_permission,
         ],
       ],
+      // Test a combination of bundles coming from different entity types.
+      [
+        [
+          'node' => ['test_group_content'],
+          'entity_test' => ['a_bundle', 'other_bundle'],
+        ],
+        array_merge($default_permissions, [
+          // node:test_group_content
+          'create test_group_content content',
+          'delete any test_group_content content',
+          'delete own test_group_content content',
+          'edit any test_group_content content',
+          'edit own test_group_content content',
+          // entity_test:a_bundle
+          'create a_bundle entity_test',
+          'delete any a_bundle entity_test',
+          'delete own a_bundle entity_test',
+          'update any a_bundle entity_test',
+          'update own a_bundle entity_test',
+          // entity_test:other_bundle
```

Comment:
226 | ERROR | [x] Inline comments must end in full-stops, exclamation marks,
     |       |     question marks, colons, or closing parentheses
```suggestion
          // entity_test:other_bundle.
```